packages/data-ingestion/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@nangohq/data-ingestion",
```

Comment:
Nice to see the first split, I think the name is debatable but no strong opinion (e.g: bigquery, bg, billing, analytics, etc.)